### PR TITLE
adds classes for responsive showing of things

### PR DESCRIPTION
**Pull request by rod**
Needed the reverse functionality of the `.hide-*` responsive classes — start hidden and show when (min width) breakpoint is reached.

Also should the hide classes be based on `max-width`?

@bellycard/apps

### CODE REVIEW

**Comment by brousalis** [comment]
for sure, I needed this functionality as well.

in my head these warrant using `max-width: ` queries. but I could be persuaded otherwise.

do me a favor and remove the index.html and compiled css. we'll recompile after this PR and do a release

**Comment by rod** [comment]
@brousalis should be good to go now

File: assets/stylesheets/rolodex/settings/utilities/lib/_show.sass [hunk 1]
```diff
@@ -0,0 +1,19 @@
+// Show
+// ========================================
+
+[class*=show-]
```
Comment (brousalis) [inline assets/stylesheets/rolodex/settings/utilities/lib/_show.sass]: lets do `[class^=show-]` here JUST incase

Comment (rod) [inline assets/stylesheets/rolodex/settings/utilities/lib/_show.sass]: [class^=show-] will only work if `show-` is the first class in the classlist

**Pull request merged**